### DISCUSSION TITLE
get row.field value in  delete Validate

### DISCUSSION THREAD

**Discussion by wow-prime** [serenity-is/Serenity#6615] [Help] [answered]
I have a field from LeftJoin and Expression.
How can I get this value in delete ValidateRequest?
Code is bellow:
![image](https://user-images.githubusercontent.com/19904096/207813201-6dba8749-a1e5-465d-afc9-390ec6ce2a2e.png)

![image](https://user-images.githubusercontent.com/19904096/207813546-dab1c979-461c-4156-be7a-51c52dbea742.png)

**Comment by edwardch** [comment]
You can try to retrieve the row data before you hit that if statement. i.e after `var user = //...`

**Comment by wezmag** [comment, marked as answer]
There is a `PrepareQuery` that can be override.
``` csharp
protected virtual void PrepareQuery(SqlQuery query)
{
    base.PrepareQuery(query);
    query.Select(MyRow.Fields.DeptNoSix);
}        
```
